src/App.tsx

```diff
@@ -1,36 +1,32 @@
 import "focus-visible/dist/focus-visible"
 import { FC } from "react"
-import { Box, ChakraProvider, Container } from "@chakra-ui/react"
+import { ChakraProvider } from "@chakra-ui/react"
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom"
 import theme from "./theme"
 import Navbar from "./components/Navbar"
 import SplashPage from "./pages/Splash"
-import Swoosh from "./static/images/Swoosh.svg"
+import useDarkMode from "./hooks/useDarkMode"
 
-const App: FC = () => {
+const App = () => {
+  useDarkMode()
+  return (
+    <>
+      <Navbar />
+      <Switch>
+        <Route exact path="/" component={SplashPage} />
+      </Switch>
+    </>
+  )
+}
+
+const ProviderWrapper: FC = () => {
```

Comment:
needed to split up the chakra provider so that the useDarkMode hook had access to the theme.